.github/workflows/run-tests.yml

```diff
@@ -4,16 +4,20 @@ on:
   push:
   pull_request:
   schedule:
-    - cron: "0 0 * * *"
+    - cron: '0 0 * * *'
 
 jobs:
   test:
     runs-on: ubuntu-latest
+
     strategy:
       fail-fast: true
       matrix:
         php: [8.2, 8.1]
-        laravel: [10.0]
+        laravel: [10.0, '11.0']
```

Comment:
Since the call to `composer require "illuminate/contracts=${{ matrix.laravel }}" ` does not have a `^` modifier on the `matrix.laravel` version number, perhaps These should be `'10.*', '11.*'` or `'^10.0', '^11.0'` instead of forcing the exact-match to `.0`?
Changes here will require a similar update to the `exclude` line below as well.